test/test_facevalues.jl

```diff
@@ -92,18 +93,18 @@ for (scalar_interpol, quad_rule) in (
 
         end
 
-        # test copy
-        fvc = copy(fv)
-        @test typeof(fv) == typeof(fvc)
-        for fname in fieldnames(typeof(fv))
-            v = getfield(fv, fname)
-            v isa Ferrite.ScalarWrapper && continue
-            vc = getfield(fvc, fname)
-            if hasmethod(pointer, Tuple{typeof(v)})
-                @test pointer(v) != pointer(vc)
-            end
-            @test v == vc
-        end
+    #     # test copy
```

Comment:
Re-enable?

Reply:
The copy test fails because of NaNs IIUS
```julia
    detJdV = fill(T(NaN), max_n_qpoints, n_faces)
```
How about changing the default to zeros?

Reply:
The reason for filling with NaN's is to catch cases when it wasn't reinit!:ed. (For example if the user forgot)

Above that test case, the fv is reinited, right?
So it should not be filled with NaN, and also not the copied version

Reply:
The thing is IIUC for pyramids and prisms not all faces has the same number of quadrature points. which makes some entries in the matrix NaNs even after init for the faces with nquadpoints <max_n_qpoints

Reply:
Aha, I'm not sure what to do here.

One option would be to fill only those entries with zero on construction to make them behave equally, but let's ask @fredrikekre on this one...

Reply:
A third option could be to just change the test to allow, add a function in the tests along the lines of 
```julia
check_equal_or_nan(a::Number, b::Number) = a==b || (isnan(a) && isnan(b))
check_equal_or_nan(a::Vector, b::Vector) = all(check_equal_or_nan, a, b)
```
This would (1) do the proper testing and (2) give out NaN if one by mistake (somehow) tries to access the uninitialized values.